cdk/drupal/drupal_stack.py

```diff
@@ -829,31 +840,30 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             "ElastiCacheClusterCacheNodeTypeParam",
             allowed_values=[ "cache.m5.large", "cache.m5.xlarge", "cache.m5.2xlarge", "cache.m5.4xlarge", "cache.m5.12xlarge", "cache.m5.24xlarge", "cache.m4.large", "cache.m4.xlarge", "cache.m4.2xlarge", "cache.m4.4xlarge", "cache.m4.10xlarge", "cache.t3.micro", "cache.t3.small", "cache.t3.medium", "cache.t2.micro", "cache.t2.small", "cache.t2.medium" ],
             default="cache.t2.micro",
-            type="String"
+            description="Required: Instance type for the memcached cluster nodes (only applies when ElastiCache enabled)."
         )
         elasticache_cluster_engine_version_param = core.CfnParameter(
             self,
             "ElastiCacheClusterEngineVersionParam",
-            # TODO: determine which versions are supported by the Drupal memcached module
             allowed_values=[ "1.4.14", "1.4.24", "1.4.33", "1.4.34", "1.4.5", "1.5.10", "1.5.16" ],
             default="1.5.16",
-            description="The memcached version of the cache cluster.",
-            type="String"
+            description="Required: The memcached version of the cache cluster (only applies when ElastiCache enabled)."
         )
         elasticache_cluster_num_cache_nodes_param = core.CfnParameter(
             self,
             "ElastiCacheClusterNumCacheNodesParam",
             default=2,
-            description="The number of cache nodes in the memcached cluster.",
+            description="Required: The number of cache nodes in the memcached cluster (only applies ElastiCache enabled).",
             min_value=1,
             max_value=20,
             type="Number"
         )
         elasticache_enable_param = core.CfnParameter(
             self,
-            "ElastiCacheEnable",
+            "_ElastiCacheEnable",
```

Comment:
Is there a reason this name in particular is prefaced with _ ?

Reply:
Good catch - I was playing with logical ids to try to get the ordering right but turns out it wasn't necessary.